File: kotlin-spring/src/main/kotlin/traveltimeplatform/models/ResponseTravelTimeStatistics.kt
package traveltimeplatform.models

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size

/**
 * 
 * @param min 
 * @param max 
 * @param mean 
 * @param median 
 */
data class ResponseTravelTimeStatistics (

        @get:NotNull 
        @JsonProperty("min") val min: Int,

        @get:NotNull 
        @JsonProperty("max") val max: Int,

        @get:NotNull 
        @JsonProperty("mean") val mean: Int,

        @get:NotNull 
        @JsonProperty("median") val median: Int
) {

}

File: php-ze-ph/src/App/DTO/ResponseTravelTimeStatistics.php
<?php

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class ResponseTravelTimeStatistics
{
    /**
     * @DTA\Data(field="min")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $min;
    /**
     * @DTA\Data(field="max")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $max;
    /**
     * @DTA\Data(field="mean")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $mean;
    /**
     * @DTA\Data(field="median")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $median;
}
